{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/utilities.py

```diff
@@ -77,3 +79,74 @@ def read_data_by_draw(artifact_path: str, key : str, draw: int) -> pd.DataFrame:
     data = pivot_categorical(data)
     data[project_globals.LBWSG_MISSING_CATEGORY.CAT] = project_globals.LBWSG_MISSING_CATEGORY.EXPOSURE
     return data
+
+
+def get_norm(
+        mean: float,
+        sd: float = None,
+        ninety_five_pct_confidence_interval: Tuple[float, float] = None
+) -> stats.norm:
+    sd = _get_standard_deviation(sd, ninety_five_pct_confidence_interval)
+    return stats.norm(loc=mean, scale=sd)
+
+
+def get_truncnorm(
+        mean: float,
+        sd: float = None,
+        ninety_five_pct_confidence_interval: Tuple[float, float] = None,
+        lower_clip: float = 0.0,
+        upper_clip: float = 1.0
+) -> stats.norm:
+    sd = _get_standard_deviation(ninety_five_pct_confidence_interval, sd)
```

Comment:
You should make the same checks here about the mean and the 95% interval.  I would also check that 
- the mean is within the clip bounds and fail if not
- something like 80% of the mass of the normal distribution is within the clip bounds and warn if not.  If a good chunk of the distribution is outside the bounds, we've probably not selected the right kind of distribution for the job.